packages/core/upload/tests/content-api/upload.test.e2e.js

```diff
@@ -190,6 +226,63 @@ describe('Upload plugin', () => {
     });
   });
 
+  describe('Create an entity with a component with a file', () => {
+    test('With an image', async () => {
+      const res = await rq({
+        method: 'POST',
+        url: '/todolists',
+        formData: {
+          data: '{"title":"Test todolist title","todo":[{"task":"First todo"},{"task":"Second todo"}]}',
+          'files.todo.0.docs': fs.createReadStream(path.join(__dirname, '../utils/rec.jpg')),
+        },
+      });
+
+      expect(res.statusCode).toBe(200);
+      expect(res.body).toMatchObject({
+        data: {
+          attributes: {
+            title: 'Test todolist title',
+          },
+          id: expect.anything(),
+        },
+      });
+      const newlyCreatedTodolist = await rq({
+        method: 'GET',
+        url: `/todolists/${res.body.data.id}`,
+        qs: {
+          populate: ['todo', 'todo.docs'],
+        },
+      });
```

Comment:
If you want you can also put the `populate` param on the POST request to have the response populated so you don't have to use `newlyCreatedTodolist`.
But if it was intended (to test that the changes were correctly made up to the GET I guess) it's perfect too 👌

Reply:
I didn't know that you could add the populate parameter in the entity creation. I guess that it's also a way to achieve the test.
I don't know if it's better to have only the POST call in the test or if adding the GET adds unwanted complexity to the test.

Reply:
I don't have a strong opinion on that. I just wanted to let you know. As you wish :)